app/src/main/java/org/supla/android/NavigationActivity.java

```diff
@@ -152,20 +153,23 @@ private RelativeLayout getMenuItemsLayout() {
             HelpButton = MenuItemsLayout.findViewById(R.id.btn_help);
             HomepageButton = MenuItemsLayout.findViewById(R.id.btn_homepage);
             AddDeviceButton = MenuItemsLayout.findViewById(R.id.btn_add);
+            EmptySpaceButton = MenuItemsLayout.findViewById(R.id.btn_empty_space);
 
             SettingsButton.setOnClickListener(this);
             AboutButton.setOnClickListener(this);
             DonateButton.setOnClickListener(this);
             HelpButton.setOnClickListener(this);
             HomepageButton.setOnClickListener(this);
             AddDeviceButton.setOnClickListener(this);
+            EmptySpaceButton.setOnClickListener(this);
```

Comment:
Is this really needed if onClick method do nothing for this button ?

Reply:
Yes, otherwise @Override onClick will not recognize that this button was clicked.

Reply:
ok, I was fooled by the empty code in onClick. You probably meant about https://github.com/SUPLA/supla-android/blob/171d9e5ec3514214607e553e62902dd54078c6da/app/src/main/java/org/supla/android/NavigationActivity.java#L408

Reply:
Yep.